turbinia/workers/analysis/wordpress.py

```diff
@@ -0,0 +1,113 @@
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Task for analysing Wordpress access logs."""
+
+from __future__ import unicode_literals
+
+import os
+import re
+
+from turbinia.evidence import ReportText
+from turbinia.workers import TurbiniaTask
+
+
+class WordpressAccessLogAnalysisTask(TurbiniaTask):
+  """Task to analyze Wordpress access logs."""
+
+  timestamp_regex = re.compile(r'\[(?P<timestamp>.+)\]')
+
+  install_step_regex = re.compile(
+      r'POST /wp-admin/install\.php\?step=2', re.IGNORECASE)
+  theme_editor_regex = re.compile(
+      r'GET /wp-admin/theme-editor\.php\?file=(?P<edited_file>.+\.php)',
+      re.IGNORECASE)
+
+  def run(self, evidence, result):
+    """Run the Wordpress access log analysis worker.
+
+    Args:
+       evidence (Evidence object):  The evidence to process
+       result (TurbiniaTaskResult): The object to place task results into.
+
+    Returns:
+      TurbiniaTaskResult object.
+    """
+    # What type of evidence we should output.
+    output_evidence = ReportText()
+
+    # Where to store the resulting output file.
+    output_file_name = 'wp_acces_log_analysis.txt'
+    output_file_path = os.path.join(self.output_dir, output_file_name)
+    # Set the output file as the data source for the output evidence.
+    output_evidence.local_path = output_file_path
+
+    # Read the input file
+    with open(evidence.local_path, 'r') as input_file:
+      wp_config_content = input_file.read()
+
+    analysis = self.analyze_wp_access_logs(wp_config_content)
+    output_evidence.text_data = analysis
+
+    # Write the report to the output file.
+    with open(output_file_path, 'w') as fh:
+      fh.write(output_evidence.text_data.encode('utf-8'))
+
+    # Add the resulting evidence to the result object.
+    result.add_evidence(output_evidence, evidence.config)
+    status = analysis.split('\n')[0]
+    result.close(self, success=True, status=status)
+    return result
+
+  def _get_timestamp(self, log_line):
+    """Extracts a timestamp from an access log line."""
+    match = self.timestamp_regex.search(log_line)
+    if match:
+      return match.group('timestamp')
+    return '[N/A]'
+
+  def analyze_wp_access_logs(self, config):
+    """Analyses access logs containing Wordpress traffic.
+
+    Args:
+      config (str): access log file content.
+
+    Returns:
+      str: Activity summary of the wordpress installation.
+    """
+    findings = []
+    findings_summary = set()
+
+    for log_line in config.split('\n'):
+
+      if self.install_step_regex.search(log_line):
```

Comment:
Super minor nit:  I think it's a little more idiomatic (and one less search()) to say:
```
match = self.foo_regex.search(bar)
if match:
  blah = match.group('foo')
```
Not sure if it's worth changing though.

Reply:
Changed (the less `search`s the better!)